deploy/v1.4.0/mainnet.ts

```diff
@@ -13,13 +13,13 @@ export default async function deployToEthereumMainnet(hre: HardhatRuntimeEnviron
     const beaconTimelockTrigger = await hre.ethers.getContract('BeaconTimelockTrigger');
     const prizeTierHistory = await hre.ethers.getContract('PrizeTierHistory');
     const lastPrizeTier = await prizeTierHistory.getPrizeTier(await(prizeTierHistory.getNewestDrawId()));
-    await deployAndLog('PrizeTierHistory', {
-        from: deployer,
-        args: [deployer],
-        skipIfAlreadyDeployed: false,
-    });
+    // await deployAndLog('PrizeTierHistory', {
+    //     from: deployer,
+    //     args: [deployer],
+    //     skipIfAlreadyDeployed: true,
+    // });
     const prizeTierHistoryNew = await hre.ethers.getContract('PrizeTierHistory');
-    await prizeTierHistoryNew.push(lastPrizeTier)
+    // await prizeTierHistoryNew.push(lastPrizeTier)
```

Comment:
It is needed @PierrickGT. The deployment had to be rerun after network issues half way through.

Reply:
I think you meant to respond to the `blockGasLimit` parameter. 
The problem with hardcoding it, is that first, it's going to be applied to all deployments, not only 1.4.0. Second, it will also be applied to all future deployments.
For example when I've deployed the TWAB Delegator contract, the gas price was set at 80. It was not needed at the time of deployment since the gas price was lower. We could have spared a few ETH in deployment cost.
I would recommend to setup an optional env variable for these kind of edge cases but not hardcode values.